doc/cephfs/lazyio.rst

```diff
@@ -21,3 +21,59 @@ LazyIO can be enabled by following ways.
 
 - ``ioctl(fd, CEPH_IOC_LAZYIO, 1UL)`` enables LAZY_IO for file handle in
    ceph-fuse mount.
```

Comment:
While you're in here, can you remove the above blurb about ceph-fuse and CEPH_IOC_LAZYIO? I don't think that is actually wired up in any fashion.

Reply:
Done!